sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTestBase.java

```diff
@@ -392,7 +392,7 @@ void createOctKeyRunner(Consumer<CreateOctKeyOptions> testRunner) {
     }
 
     void getRandomBytesRunner(Consumer<Integer> testRunner) {
-        int amountOfBytes = 16;
+        int amountOfBytes = 12;
```

Comment:
Nit: in tests I'd also use the word "count" everywhere just so it's obvious how it correlates.

Reply:
Oh true, thanks for catching that.